google/auth/impersonated_credentials.py

```diff
@@ -219,6 +227,12 @@ def __init__(
                 and self._source_credentials._always_use_jwt_access
             ):
                 self._source_credentials._create_self_signed_jwt(None)
+        if (
```

Comment:
can skip this check once the universe_domain in parameter is removed.

Reply:
removed